### PR TITLE
Corrects documentation syntax across the package

### LINKED ISSUE #163: Documentation syntax needs to be corrected across the package
As pointed out by @ben18785 in #158 (see https://github.com/epiverse-trace/serofoi/pull/158#discussion_r1492717211), the documentation of the functions can be stated in a clearer and less redundant way. I'm opening a separate issue for this since it affects other functions other than the data simulation functions.

### PR DESCRIPTION
**Pull request by ntorresd**
This PR:
- closes #163.
- correct some minor syntax problems I identified along the way
- replace `info` by `info_table` in `get_info_table`

@ben18785 please point me out to any other syntax changes you are aware of that need to be done.

### CODE REVIEW

File: R/modelling.R [hunk 1]
```diff
@@ -110,10 +110,10 @@ validate_prepared_serodata <- function(serodata) {
   stop_if_wrong_type(serodata, col_types)
 }
 
-#' Function that runs the specified stan model for the Force-of-Infection and
+#' Run specified stan model for the force-of-infection and
 #' estimates the seroprevalence based on the result of the fit
```
Comment (ben18785) [inline R/modelling.R:114]: estimate

File: R/modelling.R [hunk 2]
```diff
@@ -306,7 +307,7 @@ fit_seromodel <- function(
 }
 
 
-#' Function that generates a data.frame containing the age of each cohort
+#' Generate data.frame containing the age of each cohort
```
Comment (ben18785) [inline R/modelling.R:310]: data frame (I think generally we should stick to proper language where possible)

File: R/modelling.R [hunk 3]
```diff
@@ -484,13 +485,13 @@ extract_seromodel_summary <- function(seromodel_object,
 
 # TODO Complete @param documentation
 
-#' Function that generates an object containing the confidence interval based on
-#' a Force-of-Infection fitting
+#' Generate object containing the confidence interval based on
```
Comment (ben18785) [inline R/modelling.R:488]: What sort of object?

File: R/visualisation.R [hunk 4]
```diff
@@ -328,10 +328,8 @@ plot_rhats <- function(seromodel_object,
   return(rhats_plot)
 }
 
-# TODO Complete @param documentation
-
-#' Function that generates a vertical arrange of plots showing a summary of a
-#' model, the estimated seroprevalence, the Force-of-Infection fit and the R-hat
+#' Generate vertical arrange of plots showing a summary of a
```
Comment (ben18785) [inline R/visualisation.R:331]: Do you mean an "arrangement" of plots?

**Review by ben18785 — CHANGES_REQUESTED** [review]
A couple of minor suggested changes but looks great -- thanks @ntorresd for doing this

File: R/modelling.R (hunk 2 above)
Comment (ntorresd) [reply R/modelling.R]: Thanks, I just corrected this across the package documentation.

File: R/modelling.R (hunk 3 above)
Comment (ntorresd) [reply R/modelling.R]: A data frame as well

File: R/modelling.R (hunk 2 above)
Comment (ntorresd) [reply R/modelling.R]: Done

File: R/modelling.R (hunk 1 above)
Comment (ntorresd) [reply R/modelling.R]: Done

File: R/modelling.R [hunk 5]
```diff
@@ -389,7 +390,7 @@ get_foi_central_estimates <- function(seromodel_object,
     upper_quantile <- 0.95
     medianv_quantile <- 0.5
   }
-  # extracts foi from stan fit
+  # extracts force-of-infection from stan fit
```
Comment (ben18785) [inline R/modelling.R:393]: I would say this is a redundant comment and can be removed

**Review by ben18785 — APPROVED** [review]
Looks good @ntorresd -- I put one minor comment in here but feel free to ignore

**Pull request merged**